src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
Linter does not like static properties. You can create this property like below:

`Animal.alive = []`

this property must now be moved outside of the Animal class